src/dev_local_run.py

```diff
@@ -32,6 +30,11 @@
     "PRINT_METRIC_INGEST_INPUT": "FALSE",
 }
 
+for key, value in env_vars.items():
+    os.environ[key] = value
+
+import run_docker
```

Comment:
Yeah, I had a problem with that. Because importing at the beginning loads code from run_docker (and from subsequent imports too) first, and some global variables are set from env variables. So setting here the env vars after the import didn't take effect. 
Probably the approach in the main files could be changed, but since this is just a file to test locally, I wanted some quick fix.

Reply:
I would add a comment about the reason,  to document this 'fix'  😛